backend/src/packages/users/user.service.ts

```diff
@@ -45,11 +45,7 @@ class UserService implements Service {
   ): Promise<ReturnType<UserEntity['toObject']> | null> {
     const user = await this.userRepository.findById(id);
 
-    if (!user) {
-      return null;
-    }
-
-    return user.toObject();
+    return user?.toObject() ?? null;
```

Comment:
We should throw error here, instead of returning null

Reply:
But if we throw error here we will never throw AuthError in case when user with such email doesn't exists. Or maybe I am wrong?

Reply:
I tried it in my code and it crashes my app event if I'm catching it

Reply:
ok, got it, let's leave it then